src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/ObservationMapper.java

```diff
@@ -1,18 +1,140 @@
 package uk.nhs.digital.nhsconnect.lab.results.translator.mapper;
 
+import lombok.AllArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
 import org.hl7.fhir.dstu3.model.Observation;
+import org.hl7.fhir.dstu3.model.Observation.ObservationReferenceRangeComponent;
+import org.hl7.fhir.dstu3.model.Observation.ObservationStatus;
+import org.hl7.fhir.dstu3.model.Quantity;
+import org.hl7.fhir.dstu3.model.Quantity.QuantityComparator;
+import org.hl7.fhir.dstu3.model.SimpleQuantity;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.FreeTextSegment;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.MeasurementValueComparator;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.TestStatus;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.TestStatusCode;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.InvestigationSubject;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.LabResult;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup.ResultReferenceRange;
+import uk.nhs.digital.nhsconnect.lab.results.utils.UUIDGenerator;
 
+import java.math.BigDecimal;
 import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Component
+@AllArgsConstructor(onConstructor_ = @Autowired)
```

Comment:
Not major, but usually we use the `RequiredArgsConstructor` decorator here for constructors (as they're `final`) so may as well make it consistent

Reply:
Sure

Reply:
@andrewwa-kainos I'm happy to approve once this has been resolved